src/Workspaces/Core/Portable/Workspace/Solution/Solution.cs

```diff
@@ -1058,12 +1083,12 @@ public Solution WithDocumentSourceCodeKind(DocumentId documentId, SourceCodeKind
         /// </summary>
         public Solution WithDocumentTextLoader(DocumentId documentId, TextLoader loader, PreservationMode mode)
         {
-            return WithDocumentTextLoader(documentId, loader, textOpt: null, mode: mode);
+            return WithDocumentTextLoader(documentId, loader, text: null, mode: mode);
         }
 
-        internal Solution WithDocumentTextLoader(DocumentId documentId, TextLoader loader, SourceText textOpt, PreservationMode mode)
+        internal Solution WithDocumentTextLoader(DocumentId documentId, TextLoader loader, SourceText? text, PreservationMode mode)
```

Comment:
As a rule I'm going to remove "Opt" suffixes when the type is now annotated to that effect.